src/components/VAudioTrack/VPlayPause.vue

```diff
@@ -36,22 +36,20 @@ export default defineComponent({
      * the current play status of the audio
      */
     status: {
-      type: String,
-      validator: (val) => ['playing', 'paused', 'played'].includes(val),
+      type: String as PropType<'playing' | 'paused' | 'played'>,
```

Comment:
We could use the type from active media store here: 
https://github.com/WordPress/openverse-frontend/blob/bccbd1333fd0deb51fdd90a391d65e4666caf554/src/stores/active-media.ts#L7

Reply:
It's a different status, this one has "played" instead of "ejected". IIRC there _is_ some distinction, perhaps one for when the player is closed ("ejected") and the other for when the audio has simply finished playing to the end (in which case we show the replay button).

Reply:
Oh, thank you for explaining :)